### PR TITLE
asadiqbal08/Move prerequisites

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots

#### What are the relevant tickets?
fixes: #125
fixes: #137 

https://projects.invisionapp.com/share/RN11DSJOGXFD#/screens/455477372_Course_Detail

#### What's this PR do?
Move the prerequisites area in page.

#### Screenshots (if appropriate)
screencapture-localhost-8013-cms-pages-10-edit-preview-2021-09-03-16_29_53

screencapture-localhost-8013-cms-pages-10-edit-preview-2021-09-03-16_46_44

### CODE REVIEW

**Comment by pdpinch** [comment]
Can I get a larger screenshot of the desktop view, which includes the 4 pieces of metada, "start date", "length" "effort", and "price" ?

What happened to "price" in the mobile screenshot?

File: cms/templates/product_page.html [hunk 1]
```diff
@@ -121,6 +113,10 @@ <h1>{{ page.title }}</h1>
               <h2>About this course</h2>
               <p>{{ page.about |richtext }}</p>
             </div>
+            {% if page.prerequisites %}
+              <h2>Prerequisites</h2>
+              <p>{{ page.prerequisites |richtext }}</p>
```
Comment (arslanashraf7) [inline cms/templates/product_page.html:118]: We don't need to use an external paragraph element, the `richtext` tag automatically adds the external `<p>` tag for the whole content.
```suggestion
              {{ page.prerequisites |richtext }}
```

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]
I requested a small change. Also, it would be a good idea to rebase this PR because:

- Another PR was merged which did few tweaks to the product page as well
- The [Invision Design](https://projects.invisionapp.com/share/RN11DSJOGXFD#/screens/455477372_Course_Detail) attached to the PR has more spacing on the headers.

Doing the above might save us some regression issues.

**Comment by asadiqbal08** [comment]
@pdpinch kindly take a look and let me know do you require to modify something in design with the tiles or ok with that ? 


screencapture-localhost-8013-cms-pages-10-edit-preview-2021-09-01-16_30_57



screencapture-localhost-8013-cms-pages-10-edit-preview-2021-09-01-16_30_39

**Comment by arslanashraf7** [comment]
@asadiqbal08 I believe there is a notable difference in the spacing of the titles. Can you verify them with the [Invision Link](https://projects.invisionapp.com/share/RN11DSJOGXFD#/screens/455477372_Course_Detail)? You might also need to verify the spacing for the mobile screen. I'm attaching the screenshot from the invasion desktop design below:
Screenshot 2021-09-01 at 5 10 13 PM

**Comment by pdpinch** [comment]
A couple things:

- Arslan's screenshot definitely looks wrong. I'm not sure why it's different from yours.  
- The designer just updated the screens to standardize the background and make the info cards (start date/length/effort/price) stand out more. Can you add those changes to this PR?
- Can you also fix this bug: #137

**Comment by pdpinch** [comment]
And one more thing:

4. Please move Prerequisites down, below "What you'll learn" and above "Meet your Instructors." I'll add a comment on InVision to do that as well.

File: cms/templates/product_page.html [hunk 2]
```diff
@@ -121,6 +113,10 @@ <h1>{{ page.title }}</h1>
               <h2>About this course</h2>
               <p>{{ page.about |richtext }}</p>
             </div>
+            {% if page.prerequisites %}
+              <h2>Prerequisites</h2>
+              {{ page.prerequisites |richtext }}
+            {% endif %}
```
[the remark this answers is not in the document]
Comment (arslanashraf7) [reply cms/templates/product_page.html:119]: @asadiqbal08 I believe you need to rebase on main as I mentioned in my earlier comment on the PR, Wagtail already wraps the rich text in an external parent `<p` tag and I've removed the extra `<p` tags in one of my PRs. https://github.com/mitodl/mitxonline/pull/98/files.

**Comment by asadiqbal08** [comment]
@pdpinch I have updated the Screenshots in the description of the PR. Please take a look and let me know.

**Comment by pdpinch** [comment]
Screenshots look good. 

I think there's some overlap between this PR and https://github.com/mitodl/mitxonline/pull/146#pullrequestreview-745955300 -- please coordinate with @Wassaf-Shahzad so we avoid any regressions.

**Review by pdpinch — APPROVED** [review]

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍. One last minor comment, Feel free to merge after that.

- Since you changed property color to white, Can you remove the background color from the product detail page https://github.com/mitodl/mitxonline/blob/main/static/scss/product-details.scss#L4 as well, so that the product detail page has a design specific grey background.

**Pull request merged**